integrations/templates/setup.md

```diff
@@ -51,6 +51,8 @@ sudo ./edit-config [[ entry.setup.configuration.file.name ]]
 [% for item in entry.setup.configuration.options.list %]
 | [[ item.name ]] | [[ item.description ]] | [[ item.default ]] | [[ item.required ]] |
 [% endfor %]
+
+[[ entry.setup.configuration.options.detailed_description ]]
```

Comment:
It is not `entry.setup.configuration.options.detailed_description` but every item in the `entry.setup.configuration.options.list` may have it.

Reply:
Ah, OK, so we need a second loop then over the option list, with the option name as a heading, correct?